examples/README.md

```diff
@@ -0,0 +1,15 @@
+# FruitFlow
+
+[//]: # ({# pkglts, doc)
+
+
+[//]: # (#})
+
+Architectural Model of Fruit Tree Flowering 
```

Comment:
Update the README and present examples one by one.
You may also provide Notebook examples